components/02-molecules/cta/cta.twig

```diff
@@ -1,6 +1,6 @@
 <div class="cta">
  <h2>{{ cta__heading }}</h2>
- {% include "../../01-atoms/button/button.twig" with {
+ {% include "@atoms/button/button.twig" with {
```

Comment:
This is the namespacing syntax we need to support, which Twig.js claims to also support